### DISCUSSION TITLE
ZF1-Future 1.24.0 and composer.json

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3803] [Questions & Answers] [answered]
Today a new ZF1-Future version 1.24.0 was released. In composer.json we still have 1.23.5.

- when I run the command ```composer install``` it installs 1.23.5
- when I run the command ```composer update``` ZF1-Future is not updated to version 1.24.0
- when I run the command ```composer require shardj/zf1-future``` ZF1-Future is upgraded to version 1.24.0

Can someone explain to me why the command ```composer update``` did not update the package, because in the case of others it updated them?

Do we need to update the composer.json file?

**Comment by fballiano** [comment, marked as answer]
In the composer.json we had to fix the version number, I don't remember precisely why but there was a specific reason for doing that.

To update zf1f in openmage we've to modify the composer.json

I'm not sure if dependabot will automatically create a PR in a few days or not.

**Reply by ma4nn** [reply]
The version was fixed in order to apply the patches successfully, see https://github.com/OpenMage/magento-lts/pull/3475
